### ISSUE TITLE
Define rules for TT when multiple headers are present

### ISSUE THREAD

**Issue by koto** [WICG/trusted-types#178]
Likely, if we decide to stick with CSP (#1), we'd want to apply each policy separately to align with other CSP directive. That means e.g. that the policy name list is the intersection of the lists in all headers.

For example:

```php
header('content-security-policy: trusted-types 1a 1b common');
header('content-security-policy: trusted-types 2a 2b common');
header('content-security-policy-report-only: trusted-types 3a 3b; report-uri /');
```

This header combination should only allow `common`, and reject `1a`, `1b`, `2a` and `2b`. Additionally, it will report only warn on anything that is not `3a` or `3b`, including `common`.

Currently, in Chrome the last `CSP` and `CSP-RO` header is applied. Polyfill only allows passing it a single header.

**Comment by mikesamuel** [comment]
CSP allows multiple configurations per document.
Enforcement happens when there exists a configuration that requires enforcement.
Reporting happens for each configuration that requires reporting.

Might the simplest change be to
1. Allow multiple *TrustedTypeConfiguration*s per document.
2. Change *Obtain a Trusted Type Configuration for a response* to return plural configurations
3. Change algorithms like *Create a Trusted Type Policy*, *Get Trusted Types compliant string*, and the enforce and create a violation algos  to iterate over configurations.

2 probably depends on issue #182.

**Comment by koto** [comment]
Let's hook the TTC to the Realm, not the document. This also addresses some of the vectors in #50 (such that all in-realm documents, e.g. ones from DOMParser will be bound to the same TTC).

**Comment by koto** [comment]
Closed via #191.
